contracts/ExitFormat.sol

```diff
@@ -2,14 +2,8 @@
 pragma solidity >=0.7.0;
 pragma experimental ABIEncoderV2;
 
-/**
- * @dev Interface of the ERC20 standard as defined in the EIP.
- */
-interface ERC20Interface {
-    function transfer(address recipient, uint256 amount)
-        external
-        returns (bool);
-}
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
```

Comment:
The package `@openzeppelin/contracts` should be added to `package.json`

Reply:
Added

Reply:
You need regenerate `yarn.lock` and check that in, too, please.